### ISSUE TITLE
Double-clicking some *.gr2 crashes TriExporter

### ISSUE THREAD

**Issue by fishscene** [ray/triexporter#5]
Double-clicking any *.gr2 file crashes TriExporter. 
Version: TriExporter-0.5.1.1
OS: Windows 10 x64, OS Build 17134.648

Same behavior on my laptop, but I don't have those spec's in front of me.

No errors thrown to user. Found the following in Event Viewer:
Event ID: 1001, Source: Windows Error Reporting
```
Fault bucket , type 0
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: TriExporter.exe
P2: 0.5.0.0
P3: 55346e6b
P4: granny2.dll
P5: 2.8.12.0
P6: 499b4077
P7: c0000005
P8: 00031350
P9: 
P10: 

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER6085.tmp.WERInternalMetadata.xml

These files may be available here:
C:\ProgramData\Microsoft\Windows\WER\ReportQueue\AppCrash_TriExporter.exe_122c4de0fe6492e9cab96ab4aac5ddbd5f23eeba_3269e7c4_26b15635

Analysis symbol: 
Rechecking for solution: 0
Report Id: 57bf475b-2b9f-496e-be03-ac5bac84cadf
Report Status: 100
Hashed bucket: 
Cab Guid: 0
```
-------------

Report.wer file:
```
Version=1
EventType=APPCRASH
EventTime=131982017164017679
ReportType=2
Consent=1
UploadTime=131982017196580984
ReportStatus=100
ReportIdentifier=bf76f428-0143-44c1-a79d-66805772ba54
IntegratorReportIdentifier=57bf475b-2b9f-496e-be03-ac5bac84cadf
Wow64Host=34404
Wow64Guest=332
NsAppName=TriExporter.exe
OriginalFilename=TriExporter.exe
AppSessionGuid=0000264c-0003-0075-3b03-19e3f1e4d401
TargetAppId=W:000616dd85646c645aedcaa941c9ce289f4d00001504!0000b0c66b727caec6a61c51b6039813f9e8c0123b29!TriExporter.exe
TargetAppVer=2015//04//20:03:11:39!5aa751!TriExporter.exe
BootId=4294967295
ServiceSplit=31
TargetAsId=942
IsFatal=1
Response.type=4
Sig[0].Name=Application Name
Sig[0].Value=TriExporter.exe
Sig[1].Name=Application Version
Sig[1].Value=0.5.0.0
Sig[2].Name=Application Timestamp
Sig[2].Value=55346e6b
Sig[3].Name=Fault Module Name
Sig[3].Value=granny2.dll
Sig[4].Name=Fault Module Version
Sig[4].Value=2.8.12.0
Sig[5].Name=Fault Module Timestamp
Sig[5].Value=499b4077
Sig[6].Name=Exception Code
Sig[6].Value=c0000005
Sig[7].Name=Exception Offset
Sig[7].Value=00031350
DynamicSig[1].Name=OS Version
DynamicSig[1].Value=10.0.17134.2.0.0.256.48
DynamicSig[2].Name=Locale ID
DynamicSig[2].Value=1033
DynamicSig[22].Name=Additional Information 1
DynamicSig[22].Value=2beb
DynamicSig[23].Name=Additional Information 2
DynamicSig[23].Value=2beba6fb4680d73a8c78ca7c24ccdb46
DynamicSig[24].Name=Additional Information 3
DynamicSig[24].Value=6692
DynamicSig[25].Name=Additional Information 4
DynamicSig[25].Value=66926b94ed01e6a3828d72ce41ac00ff
UI[2]=C:\Users\Redacted\Downloads\TriExporter-0.5.1.1\TriExporter.exe
LoadedModule[0]=C:\Users\Redacted\Downloads\TriExporter-0.5.1.1\TriExporter.exe
LoadedModule[1]=C:\WINDOWS\SYSTEM32\ntdll.dll
LoadedModule[2]=C:\WINDOWS\System32\KERNEL32.DLL
LoadedModule[3]=C:\WINDOWS\System32\KERNELBASE.dll
LoadedModule[4]=C:\WINDOWS\SYSTEM32\apphelp.dll
LoadedModule[5]=C:\WINDOWS\System32\USER32.dll
LoadedModule[6]=C:\WINDOWS\System32\win32u.dll
LoadedModule[7]=C:\WINDOWS\System32\GDI32.dll
LoadedModule[8]=C:\WINDOWS\System32\gdi32full.dll
LoadedModule[9]=C:\WINDOWS\SYSTEM32\WINMM.dll
LoadedModule[10]=C:\WINDOWS\SYSTEM32\d3d9.dll
LoadedModule[11]=C:\WINDOWS\System32\msvcp_win.dll
LoadedModule[12]=C:\WINDOWS\System32\msvcrt.dll
LoadedModule[13]=C:\WINDOWS\System32\ucrtbase.dll
LoadedModule[14]=C:\WINDOWS\System32\sechost.dll
LoadedModule[15]=C:\WINDOWS\SYSTEM32\WINMMBASE.dll
LoadedModule[16]=C:\WINDOWS\System32\COMDLG32.dll
LoadedModule[17]=C:\WINDOWS\System32\RPCRT4.dll
LoadedModule[18]=C:\WINDOWS\System32\cfgmgr32.dll
LoadedModule[19]=C:\WINDOWS\System32\combase.dll
LoadedModule[20]=C:\WINDOWS\System32\SspiCli.dll
LoadedModule[21]=C:\WINDOWS\System32\CRYPTBASE.dll
LoadedModule[22]=C:\WINDOWS\System32\bcryptPrimitives.dll
LoadedModule[23]=C:\WINDOWS\System32\shcore.dll
LoadedModule[24]=C:\WINDOWS\System32\SHLWAPI.dll
LoadedModule[25]=C:\WINDOWS\System32\SHELL32.dll
LoadedModule[26]=C:\WINDOWS\SYSTEM32\dwmapi.dll
LoadedModule[27]=C:\WINDOWS\System32\windows.storage.dll
LoadedModule[28]=C:\WINDOWS\System32\advapi32.dll
LoadedModule[29]=C:\WINDOWS\System32\kernel.appcore.dll
LoadedModule[30]=C:\WINDOWS\System32\profapi.dll
LoadedModule[31]=C:\WINDOWS\System32\powrprof.dll
LoadedModule[32]=C:\WINDOWS\System32\FLTLIB.DLL
LoadedModule[33]=C:\WINDOWS\System32\ole32.dll
LoadedModule[34]=C:\WINDOWS\System32\OLEAUT32.dll
LoadedModule[35]=C:\WINDOWS\WinSxS\x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.17134.648_none_42f2d7c044decfd8\COMCTL32.dll
LoadedModule[36]=C:\WINDOWS\System32\IMM32.DLL
LoadedModule[37]=C:\WINDOWS\system32\uxtheme.dll
LoadedModule[38]=C:\WINDOWS\System32\MSCTF.dll
LoadedModule[39]=C:\WINDOWS\System32\DriverStore\FileRepository\c0339148.inf_amd64_2aa98027cbb86d51\B339146\aticfx32.dll
LoadedModule[40]=C:\WINDOWS\SYSTEM32\VERSION.dll
LoadedModule[41]=C:\WINDOWS\System32\DriverStore\FileRepository\c0339148.inf_amd64_2aa98027cbb86d51\B339146\atiu9pag.dll
LoadedModule[42]=C:\WINDOWS\System32\DriverStore\FileRepository\c0339148.inf_amd64_2aa98027cbb86d51\B339146\atiumdag.dll
LoadedModule[43]=C:\WINDOWS\System32\DriverStore\FileRepository\c0339148.inf_amd64_2aa98027cbb86d51\B339146\atiumdva.dll
LoadedModule[44]=C:\WINDOWS\SYSTEM32\amdihk32.dll
LoadedModule[45]=C:\WINDOWS\System32\TextInputFramework.dll
LoadedModule[46]=C:\WINDOWS\System32\CoreUIComponents.dll
LoadedModule[47]=C:\WINDOWS\System32\CoreMessaging.dll
LoadedModule[48]=C:\WINDOWS\SYSTEM32\ntmarta.dll
LoadedModule[49]=C:\WINDOWS\SYSTEM32\wintypes.dll
LoadedModule[50]=C:\WINDOWS\SYSTEM32\WindowsCodecs.dll
LoadedModule[51]=C:\WINDOWS\SYSTEM32\PROPSYS.dll
LoadedModule[52]=C:\WINDOWS\System32\clbcatq.dll
LoadedModule[53]=C:\Windows\System32\OneCoreUAPCommonProxyStub.dll
LoadedModule[54]=C:\WINDOWS\system32\NetworkExplorer.dll
LoadedModule[55]=C:\WINDOWS\SYSTEM32\Secur32.dll
LoadedModule[56]=C:\WINDOWS\SYSTEM32\samcli.dll
LoadedModule[57]=C:\WINDOWS\SYSTEM32\SAMLIB.dll
LoadedModule[58]=C:\WINDOWS\SYSTEM32\netutils.dll
LoadedModule[59]=C:\WINDOWS\SYSTEM32\MPR.dll
LoadedModule[60]=C:\WINDOWS\System32\drprov.dll
LoadedModule[61]=C:\WINDOWS\System32\WINSTA.dll
LoadedModule[62]=C:\WINDOWS\System32\ntlanman.dll
LoadedModule[63]=C:\WINDOWS\System32\davclnt.dll
LoadedModule[64]=C:\WINDOWS\System32\DAVHLPR.dll
LoadedModule[65]=C:\WINDOWS\System32\wkscli.dll
LoadedModule[66]=C:\WINDOWS\System32\bcrypt.dll
LoadedModule[67]=C:\WINDOWS\SYSTEM32\cscapi.dll
LoadedModule[68]=C:\Windows\System32\dlnashext.dll
LoadedModule[69]=C:\Windows\System32\PlayToDevice.dll
LoadedModule[70]=C:\Windows\System32\DevDispItemProvider.dll
LoadedModule[71]=C:\WINDOWS\System32\MMDevApi.dll
LoadedModule[72]=C:\WINDOWS\System32\DEVOBJ.dll
LoadedModule[73]=C:\Windows\System32\IPHLPAPI.DLL
LoadedModule[74]=C:\WINDOWS\System32\NSI.dll
LoadedModule[75]=C:\WINDOWS\SYSTEM32\dhcpcsvc6.DLL
LoadedModule[76]=C:\WINDOWS\SYSTEM32\dhcpcsvc.DLL
LoadedModule[77]=C:\WINDOWS\System32\WS2_32.dll
LoadedModule[78]=C:\WINDOWS\SYSTEM32\WINNSI.DLL
LoadedModule[79]=C:\WINDOWS\System32\netprofm.dll
LoadedModule[80]=C:\WINDOWS\System32\npmproxy.dll
LoadedModule[81]=C:\WINDOWS\system32\wpdshext.dll
LoadedModule[82]=C:\WINDOWS\WinSxS\x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.17134.648_none_73ddcaa59f7149a6\gdiplus.dll
LoadedModule[83]=C:\Windows\System32\PortableDeviceApi.dll
LoadedModule[84]=C:\WINDOWS\System32\SETUPAPI.dll
LoadedModule[85]=C:\WINDOWS\System32\WINTRUST.dll
LoadedModule[86]=C:\WINDOWS\System32\MSASN1.dll
LoadedModule[87]=C:\WINDOWS\System32\CRYPT32.dll
LoadedModule[88]=C:\WINDOWS\system32\audiodev.dll
LoadedModule[89]=C:\WINDOWS\system32\WMVCore.DLL
LoadedModule[90]=C:\WINDOWS\system32\WMASF.DLL
LoadedModule[91]=C:\WINDOWS\system32\mfperfhelper.dll
LoadedModule[92]=C:\Windows\System32\thumbcache.dll
LoadedModule[93]=C:\WINDOWS\SYSTEM32\policymanager.dll
LoadedModule[94]=C:\WINDOWS\SYSTEM32\msvcp110_win.dll
LoadedModule[95]=C:\Users\Redacted\Downloads\TriExporter-0.5.1.1\granny2.dll
OsInfo[0].Key=vermaj
OsInfo[0].Value=10
OsInfo[1].Key=vermin
OsInfo[1].Value=0
OsInfo[2].Key=verbld
OsInfo[2].Value=17134
OsInfo[3].Key=ubr
OsInfo[3].Value=648
OsInfo[4].Key=versp
OsInfo[4].Value=0
OsInfo[5].Key=arch
OsInfo[5].Value=9
OsInfo[6].Key=lcid
OsInfo[6].Value=1033
OsInfo[7].Key=geoid
OsInfo[7].Value=244
OsInfo[8].Key=sku
OsInfo[8].Value=48
OsInfo[9].Key=domain
OsInfo[9].Value=0
OsInfo[10].Key=prodsuite
OsInfo[10].Value=256
OsInfo[11].Key=ntprodtype
OsInfo[11].Value=1
OsInfo[12].Key=platid
OsInfo[12].Value=10
OsInfo[13].Key=sr
OsInfo[13].Value=0
OsInfo[14].Key=tmsi
OsInfo[14].Value=299200
OsInfo[15].Key=osinsty
OsInfo[15].Value=3
OsInfo[16].Key=iever
OsInfo[16].Value=11.648.17134.0-11.0.115
OsInfo[17].Key=portos
OsInfo[17].Value=0
OsInfo[18].Key=ram
OsInfo[18].Value=16313
OsInfo[19].Key=svolsz
OsInfo[19].Value=464
OsInfo[20].Key=wimbt
OsInfo[20].Value=0
OsInfo[21].Key=blddt
OsInfo[21].Value=180410
OsInfo[22].Key=bldtm
OsInfo[22].Value=1804
OsInfo[23].Key=bldbrch
OsInfo[23].Value=rs4_release
OsInfo[24].Key=bldchk
OsInfo[24].Value=0
OsInfo[25].Key=wpvermaj
OsInfo[25].Value=0
OsInfo[26].Key=wpvermin
OsInfo[26].Value=0
OsInfo[27].Key=wpbuildmaj
OsInfo[27].Value=0
OsInfo[28].Key=wpbuildmin
OsInfo[28].Value=0
OsInfo[29].Key=osver
OsInfo[29].Value=10.0.17134.648.amd64fre.rs4_release.180410-1804
OsInfo[30].Key=buildflightid
OsInfo[30].Value=63EF8990-6842-4C58-89A3-E8E867F63BF4.1
OsInfo[31].Key=edition
OsInfo[31].Value=Professional
OsInfo[32].Key=ring
OsInfo[33].Key=expid
OsInfo[34].Key=containerid
OsInfo[35].Key=containertype
OsInfo[36].Key=edu
OsInfo[36].Value=0
File[0].CabName=WERInternalMetadata.xml
File[0].Path=WER6085.tmp.WERInternalMetadata.xml
File[0].Flags=327683
File[0].Type=5
File[0].Original.Path=\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER6085.tmp.WERInternalMetadata.xml
FriendlyEventName=Stopped working
ConsentKey=APPCRASH
AppName=TriExporter_sharedcache
AppPath=C:\Users\Redacted\Downloads\TriExporter-0.5.1.1\TriExporter.exe
NsPartner=windows
NsGroup=windows8
ApplicationIdentity=3919574FBC7D67587E5D3C003FDE6525
MetadataHash=-314256997
```
---------------
WER6085_tmp_WERInternalMetadata.xml : 
```
<?xml version="1.0" encoding="UTF-16"?>
<WERReportMetadata>
	<OSVersionInformation>
		<WindowsNTVersion>10.0</WindowsNTVersion>
		<Build>17134</Build>
		<Product>(0x30): Windows 10 Pro</Product>
		<Edition>Professional</Edition>
		<BuildString>17134.648.amd64fre.rs4_release.180410-1804</BuildString>
		<Revision>648</Revision>
		<Flavor>Multiprocessor Free</Flavor>
		<Architecture>X64</Architecture>
		<LCID>1033</LCID>
	</OSVersionInformation>
	<ProblemSignatures>
		<EventType>APPCRASH</EventType>
		<Parameter0>TriExporter.exe</Parameter0>
		<Parameter1>0.5.0.0</Parameter1>
		<Parameter2>55346e6b</Parameter2>
		<Parameter3>granny2.dll</Parameter3>
		<Parameter4>2.8.12.0</Parameter4>
		<Parameter5>499b4077</Parameter5>
		<Parameter6>c0000005</Parameter6>
		<Parameter7>00031350</Parameter7>
	</ProblemSignatures>
	<DynamicSignatures>
		<Parameter1>10.0.17134.2.0.0.256.48</Parameter1>
		<Parameter2>1033</Parameter2>
		<Parameter22>2beb</Parameter22>
		<Parameter23>2beba6fb4680d73a8c78ca7c24ccdb46</Parameter23>
		<Parameter24>6692</Parameter24>
		<Parameter25>66926b94ed01e6a3828d72ce41ac00ff</Parameter25>
	</DynamicSignatures>
	<SystemInformation>
		<MID>A39D4D1C-3482-42D7-BCCE-0C3D30AE3DB0</MID>
		<SystemManufacturer>To Be Filled By O.E.M.</SystemManufacturer>
		<SystemProductName>To Be Filled By O.E.M.</SystemProductName>
		<BIOSVersion>P4.80</BIOSVersion>
		<OSInstallDate>1535776082</OSInstallDate>
		<OSInstallTime>2018-09-01T04:28:00Z</OSInstallTime>
		<TimeZoneBias>08:00</TimeZoneBias>
	</SystemInformation>
	<SecureBootState>
		<UEFISecureBootEnabled>NotCapable</UEFISecureBootEnabled>
	</SecureBootState>
	<ReportInformation>
		<Guid>bf76f428-0143-44c1-a79d-66805772ba54</Guid>
		<CreationTime>2019-03-27T23:08:39Z</CreationTime>
	</ReportInformation>
</WERReportMetadata>
```

**Comment by fishscene** [comment]
*Update*. I spent all day trying to figure this out on my laptop. Then noticed the same behavior on my Desktop. 
Since I'm mostly interested in the Triglavian stuff, that's what I did most of my testing on. 
After submitting this and clicking around more, it seems most, if not all Triglavian stuff causes TriExporter to crash. Though, there may be more that also cause it to crash, such as:
Res > dx9 > model > ship > testcases > midpoly > testcase > mb2_t2a_heavy_lowdetail.gr2
(This is a testcase, so I'm not really concerned about it crashing TriExporter, it's the Triglavian Crashes that this issue report is about)
